tools/distrib/python/grpcio_tools/protoc_lib_deps.py

```diff
@@ -19,3 +19,5 @@
 
 CC_INCLUDE='third_party/protobuf/src'
 PROTO_INCLUDE='third_party/protobuf/src'
+
+# third_party/protobuf submodule: 80a37e0782d2d702d52234b62dd4b9ec74fd2c95
```

Comment:
Given that this is a python file, does it make sense to just have the commit hash be a global variable, and then you can import this file and check it

Reply:
I tried doing that first, but the challenge was importing the file with its path being read from a variable, and it seemed different things worked differently in various python versions and the existing test relied on opening the file contents for reading. This approach sounded like more robust across python versions. If we want to do that, we should probably go about hardcoding the module path.

Reply:
I see.  If we need to add a hack to have it imported, I'm fine with it as is.

Reply:
PTAL. I made it a variable but verify its presence as a raw string, not by parsing Python.

Reply:
Still LGTM